index.html

```diff
@@ -7,32 +7,69 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <title>Awesome Book</title>
-    <link href="style.css" rel="stylesheet">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"></script>
   </head>
   <body>
-    <div class="bookDisplayContainer">
+    <div class="navbar">
+      <div class="awesome">
+        <h2>Awesome Books</h2>
+      </div>
+      <div class="navlist">
+        <ul>
+          <li class="list-items"><a href="" id="list"> List</a></li>
+          <li>|</li>
+          <li class="list-items"><a href="" id="addNew">Add new</a></li>
+          <li>|</li>
+          <li class="list-items"><a href="" id="contact">Contact</a></li>
+        </ul>
+      </div>
```

Comment:
- [ ]  **Please make sure when any of the links(List, Add, Contact) are clicked, the main area changes to display only one of the three content sections.**

![Capture](https://user-images.githubusercontent.com/63972670/206280466-66b5aaa9-8b3e-40fa-b662-57618b5e29ec.PNG)
- Kindly use this wireframe as a guide:
![Capture](https://user-images.githubusercontent.com/63972670/206280902-2dee4a26-d18e-4c58-9717-2883ec40ec83.PNG)
- **Suggestion:** You can add and remove CSS classes to display or hide the content by using the 👉  [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` method.